src/qibo/models/circuit.py

```diff
@@ -449,10 +449,9 @@ def with_noise(self, noise_map: NoiseMapType):
         :ref:`How to perform noisy simulation? <noisy-example>` example.
 
         Args:
-            noise_map (dict): Dictionary that maps qubit ids to noise
-                probabilities (px, py, pz).
-                If a tuple of probabilities (px, py, pz) is given instead of
-                a dictionary, then the same probabilities will be used for all
+            noise_map (dict): list of tuples :math:`(P_{k}, p_{k})`, where
+                :math:`P_{k}` is a ``str`` representing the :math:`k`-th Pauli
+                operator, and :math:`p_{k}` is the associated probability.
                 qubits.
```

Comment:
Maybe here the sentence is not finished.

Reply:
Thank you. Already fixed it.